web/src/components/AnalyzeData.vue

```diff
@@ -0,0 +1,35 @@
+<script>
+import Vue from 'vue';
+import { analyses } from '.';
```

Comment:
imo not a good style to import from its own `index.js`. Since it is usually the place to export data for other packages

Reply:
Couldn't really come up with a valid place to put that info.  Where would you move that object to?  Perhaps in a new more localized constants.js file?

Reply:
I wouuld go for `./vis/analyses.js` .. .default export